integrations/opennms-vmware/src/main/java/org/opennms/netmgt/poller/monitors/VmwareMonitor.java

```diff
@@ -83,6 +98,16 @@
     @Override
     public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
         final boolean ignoreStandBy = getKeyedBoolean(parameters, "ignoreStandBy", false);
+
+        final List<String> severitiesToReport =
+                Splitter.on(",")
+                    .trimResults()
+                    .omitEmptyStrings()
+                    .splitToList(getKeyedString(parameters, "reportAlarms", ""))
+                    .stream()
+                    .filter(e -> VALID_VSPHERE_ALARM_STATES.contains(e))
+                    .collect(Collectors.toList());
```

Comment:
Here it is: getKeyedString(parameters, "reportAlarms", "") defaults to an empty string